package.json

```diff
@@ -7,6 +7,7 @@
     "api:local": "NODE_ENV=local node src/api-serverless/dist/index.js",
     "backend:dev": "NODE_ENV=development node build/backend.js",
     "api:dev": "NODE_ENV=development node src/api-serverless/dist/index.js",
+    "backend:prod": "NODE_ENV=production node build/backend.js",
```

Comment:
sometimes i need to call this in prod
eg when we replayed transactions - i created a new instance of backend on an ec2 and used the above to replay all transactions since it would be a long task taking much longer than lambda lifespan